### PR TITLE
Create sensors.md

### PR DESCRIPTION
**Pull request by rgopikrishnan91**
Hi @kestewart  and @bennetkl please review

### CODE REVIEW

**Review by kestewart — CHANGES_REQUESTED** [review]
Fix typo "ina" possibly should be " on a"

**Comment by kestewart** [comment]
Also, fix typo: "where"  should be "were" in Summary

File: model/Dataset/Properties/sensors.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# sensors
+
+## Summary
+
+The field describes what sensors where used for collecting the data (if any sensors were used) 
+
+## Description
+
+Each sensor and their callibration value in form Sensor:calibration is noted. If multiple sensors are used, each sensor and thier callibration value is noted ina new line
```
Comment (armintaenzertng) [inline model/Dataset/Properties/sensors.md:11]: `sensors` has arbitrary cardinality when used in the [`Dataset` class](https://github.com/spdx/spdx-3-model/blob/dataset-profile/model/Dataset/Classes/Dataset.md), so we should not discuss multiple values in this property.

**Review by bennetkl — COMMENTED** [review]
Seems like we should use map, we can discuss in today's AI sbom meeting and I can provide description in Gopi's absense

**Comment by kestewart** [comment]
Discussed in call, and we'll apply mapping.   This should not be a blocker though for merging.     After a class is identified, we'll apply in a single pass.

**Comment by goneall** [comment]
Superseded by PR #187 which includes feedback with the exception of the map which is referenced in Issue #186

**Pull request closed without merging**